platform/darwin/src/MLNStyleValue_Private.h

```diff
@@ -239,6 +240,31 @@ class MLNStyleValueTransformer {
   // Color
   void getMBGLValue(MLNColor *rawValue, mbgl::Color &mbglValue) { mbglValue = rawValue.mgl_color; }
 
+  // VariableAnchorOffsetCollection
+  void getMBGLValue(id rawValue, mbgl::VariableAnchorOffsetCollection &mbglValue) {
+    if ([rawValue isKindOfClass:[NSArray class]]) {
+      NSArray *array = (NSArray *)rawValue;
+      if (array.count % 2 != 0) {
+        [NSException raise:NSInvalidArgumentException
+                    format:@"VariableTextAnchorOffset array should have even number of elements."];
```

Comment:
Grammar: should be `VariableTextAnchorOffset array should have an even number of elements.`